packages/launch-config/src/launch-config-crud/create.ts

```diff
@@ -2,92 +2,124 @@ import { create as createStorage } from 'mem-fs';
 import { create } from 'mem-fs-editor';
 import { join, basename } from 'path';
 import { DirName } from '@sap-ux/project-access';
-import { LAUNCH_JSON_FILE } from '../types';
-import type { FioriOptions, LaunchJSON, UpdateWorkspaceFolderOptions, DebugOptions } from '../types';
+import { LAUNCH_JSON_FILE, ProjectDataSourceType } from '../types';
+import type { FioriOptions, LaunchJSON, UpdateWorkspaceFolderOptions, DebugOptions, LaunchConfig } from '../types';
 import type { Editor } from 'mem-fs-editor';
 import { generateNewFioriLaunchConfig } from './utils';
 import { updateLaunchJSON } from './writer';
 import { parse } from 'jsonc-parser';
 import { handleWorkspaceConfig } from '../debug-config/workspaceManager';
 import { configureLaunchJsonFile } from '../debug-config/config';
-import { getFioriToolsDirectory } from '@sap-ux/store';
 import type { Logger } from '@sap-ux/logger';
-import { DatasourceType } from '@sap-ux/odata-service-inquirer';
 import { t } from '../i18n';
-import fs from 'fs';
 
 /**
- * Enhance or create the launch.json file with new launch config.
+ * Writes the `launch.json` file with the specified configurations. If the file already exists, it will be overwritten.
  *
- * @param rootFolder - workspace root folder.
- * @param fioriOptions - options for the new launch config.
- * @param fs - optional, the memfs editor instance.
- * @returns memfs editor instance.
+ * @param {Editor} fs - The file system editor used to write the `launch.json` file.
+ * @param {string} launchJSONPath - The full path to the `launch.json` file.
+ * @param {LaunchConfig[]} configurations - An array of launch configurations to be included in the `launch.json` file.
+ * @returns {void}
  */
-export async function createLaunchConfig(rootFolder: string, fioriOptions: FioriOptions, fs?: Editor): Promise<Editor> {
-    if (!fs) {
-        fs = create(createStorage());
-    }
-    const launchJSONPath = join(rootFolder, DirName.VSCode, LAUNCH_JSON_FILE);
-    if (fs.exists(launchJSONPath)) {
+function writeLaunchJsonFile(fs: Editor, launchJSONPath: string, configurations: LaunchConfig[]): void {
+    const newLaunchJSONContent = { version: '0.2.0', configurations };
+    fs.write(launchJSONPath, JSON.stringify(newLaunchJSONContent, null, 4));
+}
+
+/**
+ * Constructs the full path to the `launch.json` file based on the provided root folder.
+ *
+ * @param {string} rootFolder - The root directory where the `.vscode` folder is located.
+ * @returns {string} - The full path to the `launch.json` file.
+ */
+function getLaunchJsonPath(rootFolder: string): string {
```

Comment:
Could you maybe try to reuse `getLaunchJSONFilePath` from read.ts instead?

Reply:
As discussed, maybe its ok to just use the path as it is?